cmd/agency.go

```diff
@@ -132,7 +131,6 @@ func init() {
 	aCmd.VersionInfo = "findy-agent v" + utils.Version
 
 	flags := startAgencyCmd.Flags()
-	flags.StringVar(&aCmd.APNSP12CertFile, "apns-p12-file", "", flagInfo("APNS certificate p12 file", AgencyCmd.Name(), agencyStartEnvs["apns-p12-file"]))
```

Comment:
Unrelated to this PR, but came to my mind... Should we remove also the grpc-flag, as it is now the only way to command agent and should be enabled always?

Reply:
Excellent point! Will fix it in this same PR.